src/DotNetCore.CAP/Messages/Message.cs

```diff
@@ -67,6 +67,20 @@ public static bool HasException(this Message message)
         {
             return message.Headers.ContainsKey(Headers.Exception);
         }
-    }
 
+        public static void AddOrUpdateException(this Message message, Exception ex)
+        {
+            var msg = $"{ex.GetType().Name}-->{ex}";
```

Comment:
I think we don’t need to record the exception and the stack content, it will be more troublesome to check it, only logging the exception error  message is enough

Reply:
Very agree